ivy/utils/exceptions.py

```diff
@@ -28,22 +28,22 @@ def _remove_so_log(trace):
         if ".pyx" in repr(st):
             continue
         if "<string>" in repr(st):
-            if "compiled_fn" in repr(st) and module_frame:
+            if "traced_fn" in repr(st) and module_frame:
```

Comment:
this probably needs to be `compiled_fn` until the changes are effective in the .so file

Reply:
yeah makes sense!